### DISCUSSION TITLE
Reinstall after 7-8 days

### DISCUSSION THREAD

**Discussion by Ruskaya2023** [qnblackcat/uYouPlus#950] [Q&A]
Hello everyone! Im new to this sooo.... sorry if it was already answered but i just could not find the answear here..
My problem is that every 7 -8 days or so the app just say " youtube is not available anymore" and i have to delete it.. and put it with SIDELOAD again on my phone.... is there any way to install it for ever? or just for a longer period of time?
Thank you!!

**Comment by 420-420** [comment]
What iOS are you on? Under 16 you could use troll store above 16 you could buy a apple developer account and sign for a year.

**Comment by getBoolean** [comment]
you dont need to delete the app
